pkg/ebpf/tracer_test.go

```diff
@@ -238,6 +240,91 @@ func TestPreexistingConnectionDirection(t *testing.T) {
 	doneChan <- struct{}{}
 }
 
+func TestIncomingTCPConnectionDirection(t *testing.T) {
```

Comment:
For these four similar tests, you may be able to use a table test to avoid copying the same setup across several tests.